src/components/GameList/GameListCard.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+	export let titleGame: string;
+	export let groups: string;
+	export let gameName: string;
+	export let personAmount: string;
+	export let gameSlug: string;
+	export let isHighlighted: boolean;
+</script>
+
+<article class:highlighted-card={isHighlighted}>
```

Comment:
Ik heb nu hier wat logica toegevoegd voor die highlighted card. Kun je nog aangeven waarom je die <a> tag niet in dit component had gestopt, maar in `index.vue`? Was vast een reden voor, maar ik vind dit een stuk logischer.